### PR TITLE
Feat/relateditems

### PR DESCRIPTION
**Pull request by ryand8008**
Added Jest, 'react-test-renderer', add skeleton of related items widget

### CODE REVIEW

File: client/src/components/RelatedProducts/ComparisonTable.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  text-align: left
+`;
+
+const Item2 = styled(Item1)`
+  text-align: right
+`;
+
+function Comparison() {
+  return (
+    <div>
```
Comment (theVikingMan) [inline client/src/components/RelatedProducts/ComparisonTable.jsx:14]: Is the nested div avoiding some error in react? Just wondering why the extra piece. No change needed though if it works.

**Review by theVikingMan — APPROVED** [review]

File: client/src/components/RelatedProducts/ComparisonTable.jsx (hunk 1 above)
Comment (ryand8008) [reply client/src/components/RelatedProducts/ComparisonTable.jsx:14]: That was just something I was testing out, but forgot to remove.

**Review by floridamaniac — APPROVED** [review]

File: client/src/components/RelatedProducts/Individualcard.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import sampleData from '../../sampleData';
+
+// should retrieve all related products from initial product page
+// map each individual card with all the information
+
+export default function Individualcard() {
+  return (
```
Comment (ivyhu630) [inline client/src/components/RelatedProducts/Individualcard.jsx:9]: Recommend being consistent with the order of code blocks for readability. Some of the files has return before style components some has return after.  It's all personal preferences and doesn't impact code performance, just my 2 cents.

**Review by ivyhu630 — APPROVED** [review]

File: client/src/components/RelatedProducts/Individualcard.jsx (hunk 2 above)
Comment (ryand8008) [reply client/src/components/RelatedProducts/Individualcard.jsx:9]: I'll be sure to clean up and organize my code moving forward 👍

**Pull request merged**